### ISSUE TITLE
Not able to launch the stack

### ISSUE THREAD

**Issue by yasteeShah** [brain-power/Brain-Power-Amazon-Fidgetology#15]
I tried multiple times from the blog link and from the git, but I am not able to launch the stack.
Can you please help me out with this?

**Comment by runpengl** [comment]
Could you please provide the CloudFormation logs (from the AWS Console) or the console output from your Terminal/Shell, so I can diagnose the failure reason?

**Comment by yasteeShah** [comment]
https://user-images.githubusercontent.com/38581611/65492383-6d4e7b80-dece-11e9-859e-28a9a2fc1d67.PNG

I am getting the same window for more than 45 minutes.
I tried it with both lite vesion and full version in your git post but unfortunately every time I am stuck with the same window.

**Comment by runpengl** [comment]
Do you have full admin privileges on the AWS account you are launching the stack in?

**Comment by yasteeShah** [comment]
Yes, I do have full administrative privileges yet not able to perform the launch function.

**Comment by runpengl** [comment]
It appears that AWS did a site-wide update of its online console interface.
The new CloudFormation interface is buggy and the "Launch stack" links do not work.

You have two options:
1. If you need to get this working urgently, disable the new console interface. Then visit the "Launch stack" link again using the old interface.

2. Wait for AWS to fix the bug in their new console interface. I have also just submitted a support ticket informing AWS of this bug in their interface.

**Comment by yasteeShah** [comment]
I am totally willing to work with it as soon as possible.
But I have no idea about how to disable the new console interface.
How to go with it and will it affect my other ongoing projects in my same account?

**Comment by yasteeShah** [comment]
Okay, so now I am able to create the stack successfully but as soon as I execute it, it goes to the rollback stage and finally shows the message as ROLLBACK_FAIL.
Please help me out witj this.

https://user-images.githubusercontent.com/38581611/65725403-2def7d00-e0d0-11e9-8bca-f1bb6c4dc11e.JPG

**Comment by runpengl** [comment]
Hi, I have updated the repo documentation with new links and screenshots to reflect AWS’s new online console interface.
I have upgraded the Lambda Node.js version, which was causing your stack deploy to fail. 
Please re-fetch this repository, delete all Cloudformation resources and S3 buckets associated with this stack, and attempt to relaunch the stack.

**Comment by yasteeShah** [comment]
Hello sir, I refered your updated repo documentation. I tried launching the stack once again from scratch but I found the "Access Denied" error. This may be because the S3 link provided while creating the stack is not open to all.
Can you help me with this?
https://user-images.githubusercontent.com/38581611/65793379-47053600-e183-11e9-95ec-85c0ab3cac42.PNG

**Comment by runpengl** [comment]
Sorry, the template file was private. Just fixed

**Comment by yasteeShah** [comment]
I am not able to implement it with the updates too. I tried it for last two days but not able to fix it.
Can you help me with this?
https://user-images.githubusercontent.com/38581611/65850129-28fa2a00-e36b-11e9-87a2-9486f64e0e43.JPG

**Comment by runpengl** [comment]
CloudFormation Event logs are useful in diagnosing why a stack deploy fails.
![image](https://user-images.githubusercontent.com/5514270/65903603-b9943280-e38a-11e9-8029-059a082b53a7.png)
Could you screenshot those (click on the "Events" tab), and indicate which event gave an error?

**Comment by yasteeShah** [comment]
https://user-images.githubusercontent.com/38581611/65911991-1c1d1b00-e3eb-11e9-8836-bcfe5fd76983.JPG
https://user-images.githubusercontent.com/38581611/65912002-1fb0a200-e3eb-11e9-8c34-b6773558fd7e.JPG
https://user-images.githubusercontent.com/38581611/65912008-2212fc00-e3eb-11e9-9c4f-dfc9f1c0275b.JPG

It rollbacks and deletes the failed resources.

**Comment by yasteeShah** [comment]
Hello sir,
I hope you are not busy these days. After you asked me for the screenshots, I have attached 3 images of errors occuring.
I wish to get the response from you soon.

**Comment by cyrfer** [comment]
@yasteeShah you need to DELETE the stack AND any resources that might have been leftover. This [screenshot](https://user-images.githubusercontent.com/38581611/65912002-1fb0a200-e3eb-11e9-8c34-b6773558fd7e.JPG) shows the earliest error chronologically. You can see `WebAppBucket` already exists and Cloudformation is complaining. Delete that bucket and your stack and retry.

**Comment by yskulkarni** [comment]
The S3Key for the S3ToKVSLambda function is invalid and hence stack creation is failing with an S3 Access Denied error. Please verify by trying to access the following URL: https://s3.amazonaws.com/brain-power-fidgetology-demo-bootstrap/11cf97b799cc8689b2f0bc0f6e68069b

Please fix urgently.

**Comment by escully27** [comment]
I am seeing the same error, would love a fix as urgently as possible. the s3 access is being denied for brain-power-fidgetology-demo-bootstrap/11cf97b799cc8689b2f0bc0f6e68069b

**Comment by escully27** [comment]
Are you guys not allowing open access to this project any more? If so, it is super uncool to leave the repo and ReadMe still up. Hoping to have access soon!

**Comment by guptaravik** [comment]
> I am seeing the same error, would love a fix as urgently as possible. the s3 access is being denied for brain-power-fidgetology-demo-bootstrap/11cf97b799cc8689b2f0bc0f6e68069b

same issue here as well..

**Comment by wrigsa** [comment]
> I am seeing the same error, would love a fix as urgently as possible. the s3 access is being denied for brain-power-fidgetology-demo-bootstrap/11cf97b799cc8689b2f0bc0f6e68069b

I'm also having the same issue

**Comment by fedorzh** [comment]
I think the bucket is created with wrong permissions likely.
I was able to work around this by 

1. Downloading the full template which is created when you click the Launch button.
2. Creating my own bucket in us-east-1
3. Uploading [jar file](https://github.com/brain-power/Brain-Power-Amazon-Fidgetology/blob/master/lambda/S3ToKVS/S3ToKVS.jar) to that bucket
4. Changing the CodeUri field in the template to point to the file in bucket instead of the default one
5. Launching with my template

**Comment by ghugare-priyanka** [comment]
Omg, this is still open :(
s3 access is being denied for brain-power-fidgetology-demo-bootstrap/11cf97b799cc8689b2f0bc0f6e68069b

**Comment by andrewballester** [comment]
![Screenshot - 2022-04-27T094059 156](https://user-images.githubusercontent.com/1509007/165575805-69a46615-9134-43bb-9a5d-cde38170dfb5.png)
Anyone mind giving me a hand with this?
